### PR TITLE
Update resize messenger to include the iframe container div

### PR DESCRIPTION
**Pull request by JonNorman**
## What does this change?
When ads post a message to the main page in a request to resize them, our `resize` module resizes the ad slot and the iframe itself.

However in some circumstances we also need to resize the iframe container and it doesn't hurt to do this in all cases, hence this PR enforces this behaviour.

The layout of a typical ad is:

```
<div class="ad-slot">
    <div class="iframe-container">
        <iframe class="iframe-ad">
        </iframe>
    </div>
</div>
```

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/messenger/resize.js [hunk 1]
```diff
@@ -44,18 +45,28 @@ const resize = (
...
         Object.assign(adSlot.style, styles);
         Object.assign(iframe.style, styles);
+        Object.assign(iframeContainer.style, styles);
     });
 };
 
 const init = (register: RegisterListeners) => {
     register('resize', (specs, ret, iframe) => {
         if (iframe && specs) {
+            /* there are ultimately three containers that need resizing:
```
Comment (LATaylor-guardian) [inline static/src/javascripts/projects/commercial/modules/messenger/resize.js]: Is this comment necessary? If the containing div class name was to ever change then this comment would be invalid. I'd say your code is already self documenting given you have well named variables 👍

File: static/src/javascripts/projects/commercial/modules/messenger/resize.js [hunk 2]
```diff
@@ -44,18 +45,28 @@ const resize = (
...
+            return resize(specs, iframe, iframeContainer, adSlot);
         }
     });
 };
 
 export const _ = { resize, normalise };
 
 export { init };
+
+// #${iframe.id}__container__
```
Comment (LATaylor-guardian) [inline static/src/javascripts/projects/commercial/modules/messenger/resize.js]: Why is this commented out?

**Comment by LATaylor-guardian** [comment]
looks good 👍

File: static/src/javascripts/projects/commercial/modules/messenger/resize.js (hunk 2 above)
Comment (JonNorman) [reply static/src/javascripts/projects/commercial/modules/messenger/resize.js]: Ah I forgot to delete that line - originally I was determining the id of the iframe container but the CSS selector doesn't like ids with forward slashes in the name so I decided to just use the class instead.

File: static/src/javascripts/projects/commercial/modules/messenger/resize.js (hunk 1 above)
Comment (JonNorman) [reply static/src/javascripts/projects/commercial/modules/messenger/resize.js]: Good point - I'll remove the comment.

**Pull request merged**